sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeDirectoryClient.cs

```diff
@@ -1701,11 +1754,14 @@ public virtual DataLakeDirectoryClient GetSubDirectoryClient(string subdirectory
         /// A <see cref="RequestFailedException"/> will be thrown if
         /// a failure occurs.
         /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+#pragma warning disable AZC0002
 #pragma warning disable CS0114 // Member hides inherited member; missing override keyword
         public virtual Response<PathProperties> GetProperties(
```

Comment:
NIT - can probably condense this by having this method call the new public `GetProperties` method.

e.g.
```
public virtual Response<PathProperties> GetProperties(
            DataLakeRequestConditions conditions,
            CancellationToken cancellationToken)
            => GetProperties(new DataLakePathGetPropertiesOptions(){ Conditions = conditions }, cancellationToken)
```

Also that way if there's any fixes we need to do, we only need to do it in one method, and not have to apply it to all the methods.